### DISCUSSION TITLE
Trying to add feature flags in rabbitmq 3.12.8

### DISCUSSION THREAD

**Discussion by stenafan** [rabbitmq/rabbitmq-server#11371] [Questions]
My rabbitmq 3.12.8 wont run, and complains of missing feature flags. Problem is, in order to set the flags I first need to start the service, which then wont start cause of the missing flagss...I have no idea how to solve it, might have to downgrade (which seems annoying). Tried this, but didnt work (it wont start because of the missing flags)
C:\Program Files\RabbitMQ Server\rabbitmq_server-3.12.8\sbin>rabbitmqctl enable_feature_flag stream_single_active_consumer
I'm sorry if I'm describing it poorly, new to this developer life :) Please help me, nothing seems to work...

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ.

Yes, you must downgrade, enable all feature flags, then re-upgrade. You should consider upgrading to the latest version of RabbitMQ 3.12 (3.12.14) and then to 3.13.3.

If the above process does not work, please follow up.
